layouts/partials/html-head.hbs

```diff
@@ -18,4 +18,7 @@
     <link rel="alternate" href="/{{site.locale}}/tsc-minutes.xml" title="Node.js TSC meeting minutes" type="application/rss+xml">
     <link rel="stylesheet" href="/{{site.locale}}/styles.css" media="all">
     <link rel="stylesheet" href="/static/css/prism-tomorrow.css" media="all">
+
+    <script src="//use.typekit.net/mse5tqx.js"></script>
+    <script>try{Typekit.load();}catch(e){}</script>
```

Comment:
Naw, we should be able to, I've done it before. https://github.com/Fishrock123/Searchbeam/blob/master/src/less/fonts.less